config/initializers/devise.rb

```diff
@@ -134,4 +134,12 @@
   config.sign_out_via = :get
 
   config.case_insensitive_keys = [:email]
+
+  # keep old config
+  Devise.setup do |config|
```

Comment:
@krzysiek1507 you're doing another `Devise.setup` already inside `Devise.setup` block ;)